packages/runtime-common/index.ts

```diff
@@ -117,6 +117,19 @@ export async function chooseCard<T extends Card>(
   return await chooser.chooseCard<T>(query, opts);
 }
 
+export interface CardSearch {
+  getCards(query: Query): {
+    instances: Card[];
+    isLoading: boolean;
+  };
+}
+
+export async function getCards(query: Query) {
+  let here = globalThis as any;
+  let finder: CardSearch = here._CARDSTACK_CARD_SEARCH;
+  return finder?.getCards(query);
+}
```

Comment:
Is there a reason that these have to be in runtime-common now that we have eliminated the service worker? Can they be in host? I would love to avoid having to rely on this global if we can.

Reply:
I think we still have to go through runtime-common to access a resource in host from base

Reply:
Ah, I see.